tests/unit/lambdaService.unitTest.ts

```diff
@@ -15,7 +17,8 @@ describe("When LambdaService ", () => {
 });
 
 
-describe("When LambdaService ", () => {
+describe("When LambdaService ", function() {
```

Comment:
Fwiw, this line is unneeded - could just remove everything between lines 17 and 21 (inclusive)